### DISCUSSION TITLE
How to package the source program as a lightweight exe with a directory?

### DISCUSSION THREAD

**Discussion by GiserLi157** [GeoPyTool/GeoPyTool#25] [General]
Hello! I would like to ask you how you packaged the source program into an executable file? The reason is that I have packaged the Python program as a large exe and its directory (1.6G), and I want to package it with a 200-300M exe and its directory.

**Comment by cycleuser** [comment]
That is a good idea~
However, I really got no solution for this.
Besides, I have been occupied by daily job to make a living and almost quit this area.
